### PR TITLE
Finally fixed infinispan tests. Fixed standalone mode and cache selec…

### PR DESCRIPTION
**Pull request by honza-kasik**
…tion problem. There were also added few test for cache container settings. Some test still failing due to the HAL-873.

### CODE REVIEW

**Comment by honza-kasik** [comment]
Please, do not merge yet, I noticed another mistake...

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/InvalidationCacheTestCase.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * @author mkrajcov <mkrajcov@redhat.com>
  */
 @RunWith(Arquillian.class)
-@Category(Standalone.class)
+@Category(Shared.class)
```
Comment (honza-kasik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/InvalidationCacheTestCase.java:15]: Added Shared, although the navigation for domain mode persisted from Standalone, so the navigation for Domain mode wont work...

**Comment by honza-kasik** [comment]
Updated: fixed navigation for cache containers test case.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/InvalidationCacheTestCase.java (hunk 1 above)
Comment (honza-kasik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/InvalidationCacheTestCase.java:15]: Fixed

**Comment by zebrik** [comment]
Thank you!

**Pull request merged**